.gitignore

```diff
@@ -303,7 +303,7 @@ ASALocalRun/
 /cmake-build-debug
 
 # Doxygen
-/[Dd]ocs/
+**/[Dd]oc/generated_doxygen_docs
```

Comment:
Please help me understand why we have `**` here.

Reply:
```suggestion
**/[Dd]oc/generated_doxygen_docs/
```

Reply:
double * is in case this folder `doc` is moved in the future to other place. It means, there could be any path before it (as regex)